index.html

```diff
@@ -787,8 +787,16 @@ <h2>Links</h2>
 
       <p>
         <span class="rfc2119-assertion" id="common-constraints-links-1">The following keywords are defined for the
-          HTTP profiles and MUST be supported by all consumers.</span>
+          HTTP profiles and MAY be present in compliant TDs.</span>
+	<span class="rfc2119-assertion" id="common-constraints-links-1a">Compliant consumers that have the capabilities 
+	  to handle these relation types SHOULD treat the link targets appropriately.</span>
       </p>
+      <section class="ednote">
+      <p>For consumers that can render images or documents this implies to display appropriate documentation
```

Comment:
I strongly suggest referring to https://github.com/w3c/wot-profile/issues/255#issuecomment-1305510762 for a real restriction on the rel values together with the media types.